lib/dry/system/plugins/bootsnap.rb

```diff
@@ -5,11 +5,9 @@ module System
     module Plugins
       module Bootsnap
         DEFAULT_OPTIONS = {
-          load_path_cache: true,
-          disable_trace: true,
+          load_path_cache: false,
           compile_cache_iseq: true,
           compile_cache_yaml: true,
-          autoload_paths_cache: false
         }.freeze
```

Comment:
had to update the bootsnap dependency to get CI working again, the options removed are no longer supported in the most recent version of bootsnap.